src/components/Game/Game.js

```diff
@@ -3,19 +3,106 @@ import Map from "../Map/Map";
 import Directions from "../Directions/Directions";
 import InputBox from "../InputBox/InputBox";
 import GlobalNotification from "../GlobalNotification/GlobalNotification";
-import axios from "axios";
+import config from "../../config";
+import Pusher from "pusher-js";
 
 // rendered by App
 export default class Game extends Component {
-  componentDidMount() {}
+  state = {
+    uuid: "",
+    name: "",
+    title: "",
+    description: "",
+    players: [],
+    notifications: "",
+    userInput: "",
+    error_msg: ""
+  };
+  // TO DO,
+  handleChange = e => {
+    this.setState({ userInput: e.target.value });
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+    const { userInput } = this.state;
+    config
+      .axiosWithAuth("/api/adv/say/", userInput)
+      .then(({ data }) => {
+        // TO DO,
+        // find out what we intend to do after a user has submit an input
+      })
+      .catch(err => console.log(err));
+  };
+  /**
+   * Updates the user's character placement in the game,
+   * @param: direction,, string, can only be n,e,w,s
```

Comment:
If we want to give the user flexibility to input n/N/north/North/NORTH, we could use something like this to put all inputs into uppercase and handle multiple valid inputs, checking if it's in an array of acceptable inputs:

direction = str(input(f'Where would you like to go? ')).upper()

    if direction in ('N', 'NORTH'):
        if hasattr(room[player.cur_room], f'n_to'):
            player.cur_room = room[player.cur_room].n_to.name
        else:
            print('You cannot move North from this room.')

If we're moving off of directional buttons for movement input though, ignore this suggestion.